### ISSUE TITLE
Board name in arduino/platformio?

### ISSUE THREAD

**Issue by diramazioni** [Freenove/Freenove_ESP32_S3_WROOM_Board#6]
Hello, 
I'm having difficulties finding the correct board type in arduino/platformio, AFAIK it should be  `esp32-c3-devkitm-1` but by experimentation I found `esp32s3usbotg` it's the one that works. it's that correct?

**Comment by donwade** [comment]
I agree. What is the board name. It advertises 8M PSRAM which is what I need.
I've tried ESP32-USB-OTG simple blinky can compile and run but no PSRAM option
I've tried ESP32-CAM-LCD simply hang on boot. Offers me PSRAM but as it hangs... its useless.
I've tried ESP32-S3-DEV-module, can run blinky but barfs when running with either PSRAM option enabled (see below) using freenoves supplied webcamera example.

//-------- OPI-PSRAM  enabled 
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0x1 (POWERON),boot:0x8 (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3818,len:0x109c
load:0x403c9700,len:0x4
load:0x403c9704,len:0xb50
load:0x403cc700,len:0x2fd0
entry 0x403c98ac

using PSRAM !!!!!!!
..
WiFi connected
MFN: Partition Not found
MFN: Please Set the Partition
Camera Ready! Use 'http://10.0.0.32' to connect

--------QSPI-SRAM  log -----------------

Build:Mar 27 2021
rst:0x1 (POWERON),boot:0x9 (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3818,len:0x109c
load:0x403c9700,len:0x4
load:0x403c9704,len:0xb50
load:0x403cc700,len:0x2fd0
entry 0x403c98ac
E (569) quad_psram: PSRAM ID read error: 0x00ffffff, PSRAM chip not found or not supported, or wrong PSRAM line mode

using DRAM
..
WiFi connected
[0;31mE (1834) MFN: Partition Not found[0m
[0;31mE (1835) MFN: Please Set the Partition[0m
Camera Ready! Use 'http://10.0.0.32' to connect

**Comment by Zhentao-Lin** [comment]
[env:esp32-s3-devkitc-1]
platform = espressif32
board = esp32-s3-devkitc-1
framework = arduino
upload_port = COMx
upload_speed = 2000000
upload_protocol = esptool
monitor_speed = 115200
board_build.f_cpu = 240000000L
board_build.f_flash = 80000000L
board_build.arduino.memory_type = qio_opi

build_flags = -DCORE_DEBUG_LEVEL=0
     -DBOARD_HAS_PSRAM
     ;-mfix-esp32-psram-cache-issue
     ;-mfix-esp32-psram-cache-strategy=memw
     ;-DCONFIG_SPIRAM_USE=SPIRAM_USE_MALLOC
     ;-DSPIRAM_USE_MALLOC=1
     ;-USPIRAM_USE_CAPS_ALLOC
     ;-DCONFIG_SPIRAM_MALLOC_ALWAYSINTERNAL=100
	 ;-DCONFIG_ESP32_WIFI_TX_BUFFER=Static ;when psram is enabled, tx buffer is forced static.
	 ;-DCONFIG_ESP32_WIFI_STATIC_TX_BUFFER_NUM=64
     ;-DCONFIG_WS_BUFFER_SIZE=10240
     ;-DCONFIG_LWIP_TCP_SND_BUF_DEFAULT=102400
     ;-DCONFIG_LWIP_TCP_SND_BUF_DEFAULT=10240

**Here is my ini, you can try it if it is suitable for you. We usually use Arduino. And we are not familiar with Platformio. I'm sorry. If you have any questions, please send an email ([support@freenove.com](mailto:support@freenove.com)), we may not be able to deal with github in time.**

**Comment by donwade** [comment]
my comment was actually based on Arduino and using your webcam example.
In the end, ( for arduino 2.x users) the board type for this example is 'ESP32S3 Dev Module' with PSRAM option 'OPI PSRAM'
Once compiled, the message that barfs out on reboot can be ignored regardless of how ominous it sounds :).

Thanks for the platform-io spec. I eventually migrate my arduino stuff to platform io.

These instructions for arduino and platform-io should be moved into the README for all first time users of this board so they don't wast their time just getting simple things to run.
